workspace/read-manifest/src/index.ts

```diff
@@ -30,11 +27,10 @@ async function readManifestRaw (dir: string): Promise<unknown> {
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function validateWorkspaceManifest (manifest: any): manifest is WorkspaceManifest | undefined {
```

Comment:
Removing the `any` in favor of `unknown`. The later allows us to use TypeScript itself to ensure our validation logic is correct.